src/Tizen.NUI/src/public/BaseComponents/ImageView.cs

```diff
@@ -889,12 +890,13 @@ public FittingModeType FittingMode
         {
             get
             {
-                int ret = (int)VisualFittingModeType.Fill;
+                int ret = (int)_fittingMode;
```

Comment:
```suggestion
                return ConvertVisualFittingModetoFittingMode(_fittingMode);
```

Reply:
In the current structure, the dali and NUI values may be different, so i obtained the dali value.
i think the property map of ImageView need to refactoring, and i will changed it that time

Reply:
According to 
``` c#
if(key != Visual.Property.VisualFittingMode && _fittingMode != VisualFittingModeType.Fill)
```
A default value of fittingMode is `Fill` on Dali native, 
And you set `_fittingMode = VisualFittingModeType.Fill;` as default value
So, _fittingMode value and value from dali must be same. 

Could you tell me when it could be different?

If do not same between `_fittingMode` and value from Dali native
It also horrible things.
``` c#
image.FittingMode = FittingModeType.ScaleToFill;
anotherImage.FittingMode = image.FittingMode;
```
I expect to set, `FittingModeType.ScaleToFill` on image and anotherImage, 
But according to your comment deoes not.

Reply:
To resolve this problem, I am planning a new PR for refactoring PropertyMap in ImageView.
I think we'd better check this problem in that PR.